.github/workflows/ci_recovery.yml

```diff
@@ -0,0 +1,21 @@
+name: safe-modules-recovery
+on:
+  push:
+    paths:
+      - 'modules/recovery/**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: pnpm
+      - run: |
+          pnpm install --frozen-lockfile
```

Comment:
just fyi (i'm good with being explicit), looks like `--frozen-lockfile` is default in CI environment: https://pnpm.io/cli/install#--frozen-lockfile

Reply:
Oh didn't know! I slightly prefer to be explicit (it also makes it a bit more reproducible when trying to debug "why did CI fail" on my local machine, and copy pasting commands from the workflow file).